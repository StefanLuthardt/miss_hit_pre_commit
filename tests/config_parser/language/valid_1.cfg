matlab: "latest"
octave: "latest"

matlab: "2017b"
matlab: "2018a"
matlab: "2018b"
matlab: "2019a"
matlab: "2019b"
matlab: "2020a"
matlab: "2020b"
matlab: "2021a"
matlab: "2021b"
matlab: "2022a"
# matlab: "2022b"

octave: "4.2"
octave: "4.3"
octave: "4.4"
octave: "5.1"
octave: "5.2"
octave: "6.1"
octave: "6.2"
octave: "6.3"
octave: "6.4"
octave: "7.1"
octave: "7.2"
# octave: "7.3"
