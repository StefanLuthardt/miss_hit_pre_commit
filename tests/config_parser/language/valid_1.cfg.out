MISS_HIT configuration file
  Language: MATLAB (MATLAB 2021a)
  Language: Octave (Octave 4.4)
  Language: MATLAB 2017b (MATLAB 2017b)
  Language: MATLAB 2018a (MATLAB 2017b)
  Language: MATLAB 2018b (MATLAB 2017b)
  Language: MATLAB 2019a (MATLAB 2017b)
  Language: MATLAB 2019b (MATLAB 2017b)
  Language: MATLAB 2020a (MATLAB 2017b)
  Language: MATLAB 2020b (MATLAB 2020b)
  Language: MATLAB 2021a (MATLAB 2021a)
  Language: MATLAB 2021b (MATLAB 2021a)
  Language: MATLAB 2022a (MATLAB 2021a)
  Language: Octave 4.2 (Octave 4.2)
  Language: Octave 4.3 (Octave 4.2)
  Language: Octave 4.4 (Octave 4.4)
  Language: Octave 5.1 (Octave 4.4)
  Language: Octave 5.2 (Octave 4.4)
  Language: Octave 6.1 (Octave 4.4)
  Language: Octave 6.2 (Octave 4.4)
  Language: Octave 6.3 (Octave 4.4)
  Language: Octave 6.4 (Octave 4.4)
  Language: Octave 7.1 (Octave 4.4)
  Language: Octave 7.2 (Octave 4.4)
MISS_HIT Debug Summary: 1 file(s) analysed, everything seems fine
